internal/providers/terraform/aws/registry.go

```diff
@@ -103,6 +103,7 @@ var ResourceRegistry []*schema.RegistryItem = []*schema.RegistryItem{
 	getDirectoryServiceDirectory(),
 	getTransferServerRegistryItem(),
 	getNetworkfirewallFirewallRegistryItem(),
+	getGlobalAcceleratorRegistryItem(),
 }
 
 // FreeResources grouped alphabetically
```

Comment:
Please add aws_globalaccelerator_listener to the list of free resources since there are no costs associated with it.